staker/block_validator.go

```diff
@@ -112,27 +119,21 @@ func (c *BlockValidatorConfig) Validate() error {
 		c.memoryFreeLimit = limit
 	}
 	streamsEnabled := c.RedisValidationClientConfig.Enabled()
-	if c.ValidationServerConfigs == nil {
-		c.ValidationServerConfigs = []rpcclient.ClientConfig{c.ValidationServer}
-		if c.ValidationServerConfigsList != "default" {
-			var validationServersConfigs []rpcclient.ClientConfig
-			if err := json.Unmarshal([]byte(c.ValidationServerConfigsList), &validationServersConfigs); err != nil && !streamsEnabled {
+	if c.ExecutionServerConfigs == nil {
```

Comment:
Checking `== nil` on a slice is always a bit concerning to me. Is it possible that a future koanf change breaks this? Ideally I'd like either a test or to change the check to `len(...) == 0`.